src/components/UserSelector.tsx

```diff
@@ -1,45 +1,39 @@
-import React, { useContext, useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import classNames from 'classnames';
-import { UserContext } from './UsersContext';
 import { User } from '../types/User';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
+import { loadUsers } from '../features/users/usersSlice';
+import { authorSlice } from '../features/author/autthorSlice';
```

Comment:
There is a typo in the import path for `authorSlice`. The directory name `autthorSlice` seems to be misspelled. Ensure that the path is correct to avoid import errors.